host_modules/systemd_service.py

```diff
@@ -7,6 +7,8 @@
 ALLOWED_SERVICES = ['snmp', 'swss', 'dhcp_relay', 'radv', 'restapi', 'lldp', 'sshd', 'pmon', 'rsyslog', 'telemetry']
 EXIT_FAILURE = 1
 
+REBOOTMETHOD_COLD_BOOT_VALUES = {1, "COLD"}
+REBOOTMETHOD_HALT_BOOT_VALUES = {3, "HALT"}
```

Comment:
>3 [](http://example.com/codeflow?start=33&length=1)

This service is independent with gnoi. So it may be better to use common dbus enum or similar types, if no such type, string is just okay.

Reply:
I have used enum in the code. Thank you!